test/integration/resultsPage.js

```diff
@@ -15,136 +15,216 @@ const resultsRoute = `${constants.SITE_ROOT}/results/`;
+      const search = 'Surgery';
+      chai.request(app)
+        .get(resultsRoute)
+        .query({ search })
+        .end((err, res) => {
+          const $ = cheerio.load(res.text);
 
-        done();
+          const resultsHeader = $('.results__header').text();
+          expect(resultsHeader).to.contain(`GP surgeries matching '${search}'`);
+          done();
+        });
+    });
+    describe('matching surgeries found', () => {
+      describe('single match', () => {
+        const search = 'Idle';
+        it('should have only one result', (done) => {
+          chai.request(app)
+            .get(resultsRoute)
+            .query({ search })
+            .end((err, res) => {
+              const $ = cheerio.load(res.text);
+
+              const searchResults = $('.results__item--nearby');
+              expect(searchResults.length).to.equal(1);
+
+              done();
+            });
+        });
+        it('should display singular message text', (done) => {
+          chai.request(app)
+            .get(resultsRoute)
+            .query({ search })
+            .end((err, res) => {
+              const $ = cheerio.load(res.text);
+
+              const resultsHeader = $('.results__header').text();
+              expect(resultsHeader).to.contain(`GP surgery matching '${search}'`);
+
+              done();
+            });
+        });
       });
-  });
-
-  it('should return a booking link for a surgery that has a booking supplier', (done) => {
-    const search = 'Crookes Valley Medical Centre Sheffield';
-    chai.request(app)
-      .get(resultsRoute)
-      .query({ search })
-      .end((err, res) => {
-        iExpect.htmlWith200Status(err, res);
-
-        const $ = cheerio.load(res.text);
-
-        const resultsHeader = $('.results__header').text();
-        expect(resultsHeader).to.contain(`GP surgeries matching '${search}'`);
-
-        const searchResults = $('.results__item--nearby .results__details').first();
-        expect(searchResults.html()).to.not.contain(noOnlineBookingLinkMessage);
-        expect(searchResults.html()).to.contain('href');
-
-        expect($('.link-back:first-of-type').eq(0).text()).to.equal('Back');
-        expect($('.link-back:first-of-type').eq(1).text()).to.equal('Back');
-        expect($('.link-back').attr('href')).to.equal(`${constants.SITE_ROOT}`);
-        done();
+      describe('multiple matches', () => {
+        it('should have more than one result', (done) => {
+          const search = 'Surgery';
+          chai.request(app)
+            .get(resultsRoute)
+            .query({ search })
+            .end((err, res) => {
+              const $ = cheerio.load(res.text);
+
+              const searchResults = $('.results__item--nearby');
+              expect(searchResults.length).to.be.above(1);
+
+              done();
+            });
+        });
+        it('should display plural message text', (done) => {
+          const search = 'Surgery';
+          chai.request(app)
```

Comment:
Same as previous comment about removing duplication and running the request within the `describe` block